pkg/commands/testdata/testcase-configmaps/base/myapp/mycomponent/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+namePrefix: com1-
```

Comment:
Please make this
```
namePrefix: p1-
```
and the other prefix `p2-` so it's easier to distinguish the prefix from the name